src/index.js

```diff
@@ -0,0 +1,40 @@
+document.getElementById("up-button").onclick = function(){
+    var tempInt = parseInt(document.getElementById("temperature").innerHTML);
```

Comment:
Consider introducing a state variable outside the functions to hold the temperature, and only writing it to the display element, rather than using the text in the display element _as_ the storage and needing to parse it back.